src/Microsoft.ML.Data/Transforms/NAFilter.cs

```diff
@@ -79,10 +79,11 @@ private static VersionInfo GetVersionInfo()
         private const string RegistrationName = "MissingValueFilter";
 
         /// <summary>
-        /// Convenience constructor for public facing API.
+        /// Convenience constructor for creating <see cref="NAFilter"/>.
```

Comment:
>/// [](start = 8, length = 3)

// instead of the triple

Reply:
Any specific reason for this because its a normal C# summary comment which always starts with ///

---
In reply to: [207688810](https://github.com/dotnet/machinelearning/pull/646#discussion_r207688810) [](ancestors = 207688810)